watertap/unit_models/tests/test_reverse_osmosis_0D.py

```diff
@@ -311,12 +311,7 @@ def test_calculate_scaling(self, RO_frame):
         # check that all variables have scaling factors
         unscaled_var_list = list(unscaled_variables_generator(m))
         assert len(unscaled_var_list) == 0
-        # check that all constraints have been scaled
-        unscaled_constraint_list = list(unscaled_constraints_generator(m))
```

Comment:
@bknueven since some constraints are still being scaled, should we include those in the test to verify that they are being scaled? That way, if changes are made later on, we'd have an additional layer of tracking which constraints typically benefit from scaling and which can be left alone. It might be more troublesome than worthwhile though. What do you think?

Reply:
I guess I'm still open to being more selective on scaling the constraints. If #285 goes before this PR, scaling the constraints isn't strictly necessary (at least for the tests to pass).